Make.scan-build

```diff
@@ -0,0 +1,23 @@
+include $(TOPDIR)/Make.rules
+include $(TOPDIR)/Make.defaults
```

Comment:
Maybe I'm missing something but I don't see why these includes are needed here, since all Make.* are included from the top level Makefile.